bin/python/ripple/util/Sign.py

```diff
@@ -22,9 +23,9 @@
     check <key>
         Check an existing key for validity.
 
-    sign <sequence> <validator-public> <master-secret>
+    sign <sequence> <validator-public-key> <validator-private-key> <master-secret>
         Create a new signed manifest with the given sequence
-        number, validator public key, and master secret key.
+        number, validation public key, and master public key.
```

Comment:
I totally overlooked `doc/manifest-tool-guide.md`. Sorry about that.
I currently have the manifest tool using `validation_private_key` instead of `validation_seed` primarily because I wasn't having success deriving the ecdsa keypair from the seed in python.